packages/server/lib/util/print-run.ts

```diff
@@ -274,7 +287,17 @@ export function displaySpecHeader (name: string, curr: number, total: number, es
   }
 }
 
-export function renderSummaryTable (runUrl: string | undefined, results: any) {
+function maybeLogCloudRecommendationMessage (runs: CypressCommandLine.RunResult[], record?: boolean) {
```

Comment:
Hmm what do you think either

1. make `record` non nullable
2. or have a default value

Just a personal preference, but I like for booleans to be explicit, so we can communicate the expected default value to future developers.